notebooks/5_ModelComparison/1_Ins_ModelComparison.ipynb

```diff
@@ -123,6 +116,12 @@
     "#### *Deviance*: Estimating additional uncertainty because we don't know the truth using Deviance\n",
     "If we knew the truth then there wouldn't be a point to this tutorial, or the entire field of Statistics/Machine Learning. Since we don't know the truth this unfortunately means we can't use the formula above, but through some clever thinking it turns out you don't need to know the truth to compare two models, just how far off one model is relative from another. By using the truth data (The weather in the past), and a model estimates, its possible to calculate **Deviance** which is an estimate of the divergence.\n",
     "\n",
+    "Taken from Osvaldo Martin's book [Bayesian Data Analysis in Python](https://www.packtpub.com/big-data-and-business-intelligence/bayesian-analysis-python-second-edition)\n",
+    "\n",
+    "$$ D_{kl}(p \\mid \\mid q) - D_{kl}(q \\mid \\mid r) \\approx \\sum_{n=1}^{N}(log(q_{i}) - \\sum_{n=1}^{N} log(r_{i})) $$\n",
```

Comment:
the are a couple of extra parentheses. $$D_{kl}(p \mid \mid q) - D_{kl}(q \mid \mid r) \approx \sum_{n=1}^{N} log(q_{i}) - \sum_{n=1}^{N} log(r_{i})$$

Reply:
Ah thank you. Always fixing my formulas